### ISSUE TITLE
Remove package-lock from node

### ISSUE THREAD

**Issue by tuxredux** [joeblau/gitignore.io#363]
According to the [npm docs](https://docs.npmjs.com/files/package-lock.json):
> This file is intended to be committed into source repositories, and serves various purposes:

**Comment by gluons** [comment]
This repo use template from [dvcs/gitignore](https://github.com/dvcs/gitignore).

This file has been ignored by dvcs/gitignore@d487ea2bc0c66ca2b88b9ba4795978deb9258a9d .

I've told them in dvcs/gitignore#45 . And I've sent PR dvcs/gitignore#48 to fix this.

I've been waiting it to be merged. 🙄

**Comment by tuxredux** [comment]
@gluons Sorry, I should have checked dependencies first. Great work thanks!

**Comment by gluons** [comment]
@joeblau `yarn.lock` and `package-lock.json` still appear in **gitignore.io**.

<img width="455" alt="default" src="https://user-images.githubusercontent.com/4688092/28253709-c617dc1c-6ad0-11e7-9403-ec72daaea197.PNG">

**Comment by joeblau** [comment]
It's gone now.  I updated the production app this morning.

![screen shot 2017-07-17 at 9 25 55 am](https://user-images.githubusercontent.com/1218847/28270014-fc3c0420-6ad1-11e7-8266-3a15f7bfe587.png)

I also created #364 to script updates so they push all the way to prod automatically, when a template is updated that way I just have to review the template and once it's approved, it goes live after build/test/integ.
